config/default.yml

```diff
@@ -553,6 +553,10 @@ Style/TrivialAccessors:
     - to_s
     - to_sym
 
+Style/UnneededPercentX:
+  # Enforce using %x when the command string spans multiple lines.
+  EnforcePercentXForMultiLine: false
```

Comment:
I can think of 4 different “preferred style” configurations for this “CommandLiteral” cop:

 1. `backticks_unless_has_inner_backtick`: doesn't care whether the literal is single or multi-line.
 2. `always_backticks`: doesn't care about inner backticks.
 3. `always_percent_x`: doesn't care about inner backticks.
 4. `percent_x_for_multiline`: doesn't care about the style of single-line command literals.

And I’m not sure whether I’m ok with the “don’t care”’s for 1) and 4). I might like an option where multi-lines are always `%x`, and single-lines are backtick’ed except if there’s an inner backtick.

Would it make sense to configure the styles separately for single-line and multi-line literals?